verify-push-backend/functions/access-token.js

```diff
@@ -61,10 +60,10 @@ exports.handler = function (context, event, callback) {
 
   const client = context.getTwilioClient();
   const serviceSid = context.VERIFY_SERVICE_SID;
-  const identity = crypto
-    .createHash('sha256')
-    .update(event.identity)
-    .digest('hex');
+  const hashIdentity = context.IDENTITY_PROCESSING !== 'raw';
```

Comment:
if we change the context variable to `HASH_IDENTITY=true` we can skip this line and just use the context variable on line 64